projects/core/src/lib/operators/dialogs/raster-scaling/raster-scaling.component.html

```diff
@@ -0,0 +1,78 @@
+        formulas to every pixel.
+        <br />
+        For <b> unscaling </b> the formula is: <i>p_new = p_old * slope + offset</i>. <br />
+        For <b> scaling </b> the formula is: <i>p_new = (p_old - offset) / slope</i>. <br />
+        <i>p_old</i>> and <i>p_new</i> refer to the old and new pixel values. The slope and offset values are either properties attached to
+        the input raster or a fixed value. For more information see
+        <a href="https://docs.geoengine.io/operators/rasterunscaling.html">docs.geoengine.io</a>.
+    </p>
+</geoengine-dialog-help>
+
+<form [formGroup]="form" (ngSubmit)="add()" fxLayout="column" ngNativeValidate>
+    <div fxFlex>
+        <geoengine-layer-selection [types]="inputTypes" formControlName="layer"></geoengine-layer-selection>
+        <geoengine-dialog-section-heading title="Config" subtitle="Specify the output data type"></geoengine-dialog-section-heading>
+
+        <div formGroupName="scale">
+            <mat-slide-toggle formControlName="byKey" #byKey>Use scale factor from metadata key</mat-slide-toggle>
+            <mat-form-field appearance="fill">
+                <mat-label>Scale metadata <i>domain</i></mat-label>
+                <input matInput type="text" formControlName="domain" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
+                <mat-label>Scale metadata <i>key</i></mat-label>
+                <input matInput type="text" formControlName="key" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
+                <mat-label>Scale <i>Constant</i></mat-label>
+                <input matInput type="number" formControlName="constant" />
+            </mat-form-field>
+        </div>
+
+        <div formGroupName="offset">
+            <mat-slide-toggle formControlName="byKey" #byKey>Use offset value from metadata key</mat-slide-toggle>
+            <mat-form-field appearance="fill">
+                <mat-label>Offset metadata <i>domain</i></mat-label>
+                <input matInput type="text" formControlName="domain" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
+                <mat-label>Offset metadata <i>key</i></mat-label>
+                <input matInput type="text" formControlName="key" />
+            </mat-form-field>
+            <mat-form-field appearance="fill">
+                <mat-label>Offset <i>Constant</i></mat-label>
+                <input matInput type="number" formControlName="constant" />
+            </mat-form-field>
+        </div>
+
+        <mat-form-field appearance="fill">
+            <mat-label>Scale Type</mat-label>
+            <mat-select formControlName="scaleType">
+                <mat-option *ngFor="let scaleType of scaleTypes" [value]="scaleType.type">
+                    {{ scaleType.type }} <i>{{ scaleType.formular }}</i>
```

Comment:
can you highlight the type? it looks like it's part of the formula

Reply:
i tried and i can only add styles in the listing... [1881667](https://github.com/geo-engine/geoengine-ui/pull/471/commits/18816673e6150fb0636cb4d991dfbe137b2a78de)

Reply:
[7509d42](https://github.com/geo-engine/geoengine-ui/pull/471/commits/7509d42b23055a6a657a39ac784be25f4a2d89bc)